hdk/assembly/code.py

```diff
@@ -200,3 +202,23 @@ def translate(instructions: Iterable[Instruction]) -> Iterator[str]:
             yield translate_a_instruction(instruction, symbol_table)
         else:
             yield translate_c_instruction(instruction)
+
+
+def link_instructions(instructions: Iterable[Instruction]) -> list[Command]:
+    """Translates instructions in commands for bringing variables into the table.
+
+    Args:
+        instructions: An iterable of symbolic instructions.
+
+    Returns: An list of commands
+
+    """
+    commands, symbol_table = _do_first_pass(instructions)
+    linked_commands = []
+    for command in commands:
+        if isinstance(command, AInstruction) and not command.symbol.isdigit():
+            command = AInstruction(
```

Comment:
Add a local variable address = symbol_table.get_symbol_address(command.symbol). Use it in an AInstruction construction.